packages/sdk/src/core/schema/QueryParams.ts

```diff
@@ -24,3 +24,30 @@ export type QueryAllParams = Exclude<
   z.input<typeof QueryAllParamsSchema>,
   undefined
 >;
+
+export const QueryOwnedParamsSchema = /* @__PURE__ */ (() =>
```

Comment:
just looking at this vs `QueryAllParams` - any reason why they're different? feels like we could re-use the same no?

Also if its a new type, should be exported in the `index.ts` in the same folder

Reply:
@joaquim-verges The `start` in `QueryAllParams` is meant for the tokenId. Which is useful when querying all NFTs in the collection.

However for `getOwned()` the `start` is not quite applicable since we don't know if user has the exact tokenId, so instead it should be `page` - that's the way I think about it.

I'll check the exporting part now

Reply:
Done exporting: https://github.com/thirdweb-dev/js/pull/1521/commits/a9ff52fd19647c44026e847e0f11539131a5819a

Reply:
`start` is not necessarily tokenId, its just where you want it to start from. The logic for getAll pagination is something like:
start = 0, count = 10
start = 10, count = 10
start = 20, count = 10
etc

its annoying to then switch your logic for getOwned() to
page = 0, count = 10,
page = 1, count = 10
page = 2, count = 10
...

let's keep it consistent, i think the `start` approach is fine (just take the last `start + count`). 

annoying to have 2 pagination logic in apps